manager.py

```diff
@@ -68,18 +60,7 @@ def configure(self, context):
 
         # Register dependencies to rest and block manager
         self._rest_manager = self.get_dependency("RESTManager")
-        self._block_manager = self.get_dependency("BlockManager")
-
-        # Clone down any blocks that were configured in the instance settings
-        try:
-            self._clone_configured_blocks()
-        # catch the two known exceptions that a corrupt/invalid file might throw
-        except (TypeError, ValueError):
-            # log and allow nio to continue
-            self.logger.exception(
-                "An error has occurred while making sure installed block source"
-                " files are present, please verify that 'blocks.cfg' entries"
-                " are valid")
+        self._block_cloner = BlockCloner(self.get_dependency("BlockManager"))
```

Comment:
Something feels funky about the way this is set up. The `BlockCloner` feels like it should be part of the `BlockManager` to me but then here we instantiate the `BlockCloner` directly and then pass the `BlockManager`. It might be worth thinking about this setup and discussing it a bit further.

Reply:
It does, I am making some changes to make the BlockCloner use of BlockManager transparent to calling parties